crypto/dsa/dsa_pmeth.c

```diff
@@ -178,9 +178,7 @@ static int pkey_dsa_ctrl_str(EVP_PKEY_CTX *ctx,
     }
     if (strcmp(type, "dsa_paramgen_q_bits") == 0) {
         int qbits = atoi(value);
-        return EVP_PKEY_CTX_ctrl(ctx, EVP_PKEY_DSA, EVP_PKEY_OP_PARAMGEN,
-                                 EVP_PKEY_CTRL_DSA_PARAMGEN_Q_BITS, qbits,
-                                 NULL);
+        return EVP_PKEY_CTX_set_dsa_paramgen_q_bits(ctx, qbits);
```

Comment:
I am unsure about backporting this macro.

Reply:
Considering the text of the commit, that this is a programming safety measure, I think it's ok.

Reply:
A missing ctrl wrapper macro for a ctrl code could also be considered to be analogous to a missing accessor. But as I indicated in the description, the list of backports is debatable. If Matt prefers not to have it backported, I could drop it.

Reply:
I'm ok with it. I just had a moments hesitation as to whether we were overstepping our rules with this change.